src/parser/open-api/v3/open-api-v3-parser.js

```diff
@@ -110,7 +110,9 @@ function addMediaTypeInfoToUIObject (uiObject, mediaType) {
   }
 
   if (mediaType.examples) {
-    examples = examples.concat(...mediaType.examples)
+    examples = examples.concat(...Object.keys(mediaType.examples).map(
```

Comment:
Going to recommend the spread operator over concat.

`examples = [...examples, ...Object.keys()]`

Or push

`examples.push(...Object.keys())`